src/map/script.cpp

```diff
@@ -21572,6 +21572,44 @@ BUILDIN_FUNC(instance_live_info)
 	}
 	return SCRIPT_CMD_SUCCESS;
 }
+/*==========================================
+ * set '.@instance_list' to a list of the live instance ids for the map with the mode.
+ * return the array size of '.@instance_list'
+ * .@size = instance_list(IM_NONE, "prontera");
```

Comment:
Maybe I misunderstand the usecase of this function. But I would flip the parameters and make mode optional (default party?)?
Because maybe you want to know all prontera instances from all modes?

Reply:
I changed it to return all the modes by default , and accept the map name only , if I made the party mode by default I would need to create a something outside the IM_* (0~4) to return all modes , so now if tie IM_* is not provided it will return all.

I just got this while writing this reply
should I `IM_MAX` to return all and no mode return party ?